webapp/src/pages/createForm/components/ApplicationForm.js

```diff
@@ -0,0 +1,181 @@
+import React, { useState, useEffect } from 'react';
+import { useTranslation } from 'react-i18next'
+import { eventService } from '../../../services/events';
+import icon from '../icon.svg';
+import Section from './Section';
+
+const ApplicationForm = (props) => {
+  const [nominate, setNominate] = useState(false);
+  const [event, setEvent] = useState({
+    loading: true,
+    event: null,
+    error: null,
+  });
+  const [sections, setSections] = useState([{
+    id: 1,
+    name: 'Untitled Section',
+    description: '',
+    questions: [
+      {
+        id: `${Math.random()}`,
+        order: 1,
+        headline: '',
+        placeholder: '',
+        type: null,
+        options: [],
+        value: '',
+        label: '',
+        required: false
+      }
+    ]
+  }]);
+  const { t } = useTranslation();
+
+  useEffect(() => {
+    eventService.getEvent(props.event.id).then( res => {
+      setEvent({
+        loading: res.loading,
+        event: res.event,
+        error: res.error
+      })
+    })
+  }, []);
+  const handleCheckChanged = (e) => {
+    setNominate(e.target.checked);
+  }
+  const handleSection = (input) => {
+    setSections(input)
+  }
+  const addSection = () => {
+    const id = sections.length + 1;
+    setSections([...sections, {
+      id: id,
+      name: 'Untitled Section',
+      description: '',
+      questions: [
+        {
+          id: 1,
+          order: 1,
+          headline: '',
+          placeholder: '',
+          type: null,
+          options: [],
+          value: '',
+          label: '',
+          required: false
+        }
+      ]
+    }]);
+  }
+  const addQuestion = (id) => {
```

Comment:
Nit: can you call this `sectionId` so that it's clear from the function signature that this is referring to the section and not the question